src/styles/card.scss

```diff
@@ -0,0 +1,93 @@
+.card {
+  width: $Card-width;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 32px 16px 16px;
+  background-color: white;
+  border-radius: 5px;
+  border: 1px solid $White-accent;
+  box-sizing: border-box;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    padding-bottom: 40px;
+    align-self: center;
+  }
+
+  &__logo {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 1.5em;
+    font-style: normal;
+    padding-bottom: 4px;
+  }
+
+  &__code {
+    color: $Secondary;
+    font-size: 10px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 1.4em;
+    padding-bottom: 16px;
+  }
+
+  &__review {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding-bottom: 24px;
+
+    &-text {
+      font-size: 10px;
+      font-style: normal;
+      font-weight: 400;
+      line-height: 16px;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding-bottom: 16px;
+
+    &-text {
+      color: $Secondary;
+      font-size: 12px;
+      font-style: normal;
+      font-weight: 400;
+      line-height: 18px;
+    }
+
+    &-number {
+      font-size: 16px;
+      font-style: normal;
+      font-weight: 700;
+      line-height: 18px;
+    }
+  }
+
+  &__buy {
+    padding: 11px 0;
+    background-color: $Blue-accent;
+    border: 1px solid transparent;
+    border-radius: 5px;
+    text-align: center;
+    font-size: 14px;
+    font-style: normal;
+    font-weight: 700;
+    line-height: 16px;
+    text-transform: uppercase;
+    color: white;
+
+    &:hover {
+      cursor: pointer;
+      background: transparent;
+      border-radius: 5px;
```

Comment:
There is no need to duplicate properties that are the same for the default state of the element